flake.nix

```
@@ -46,18 +46,22 @@
       });
   in {
     hjemModules = {
-      hjem-rum = import ./modules/hjem.nix {
+      complete = import ./modules/hjem.nix {
         inherit (nixpkgs) lib;
         inherit rumLib;
       };
-      default = self.hjemModules.hjem-rum;
+      bare = {_module.args = {inherit rumLib;};};
```

```suggestion
      hjemModules = let
        dependencies = { _module.args = {inherit rumLib;}'};
        in
          complete = import ./modules/hjem.nix {
            inherit (nixpkgs) lib;
            } // dependencies;
            inherit dependencies;
```

I  think that bare is slightly misleading. What do you think about doing it like above? This is clearer imo, and allows us to extend it more easily if needed (the usage would stay the exact same).

This way, the user knows what they're importing and is aware of what side-effects this will have.

About the idea itself,  I guess I'm not a fan of messing with hjem's `_module.args`, but this is the only solution I can think of at this moment.